executables/collector/src/main/java/de/cyface/collector/verticle/CollectorApiVerticle.java

```diff
@@ -329,8 +329,8 @@ private void addAuthenticatedPostHandler(Router router, JWTAuth jwtAuth,
         router.post(endpoint)
                 .consumes("application/json; charset=UTF-8")
                 // Ready request body only once and before async calls or pause/resume must be used see [DAT-749]
-                .handler(bodyHandler)
                 .handler(LoggerHandler.create())
+                .handler(bodyHandler)
```

Comment:
At least the comment is misplaced now, as it should stick with `bodyHandler`.

The comment indicates, that no handlers which use `request.pause/resume` (to wait for a promise) may be called before registering `bodyHandler`. https://vertx.io/docs/vertx-web/java/#_request_body_handling "requester this hander as soon as possible". 

I guess the LoggingHandler from Vert.X might be OK to be called before, we'll see? The tests seems to accept this.

Please move at least the comment to the right position.

Reply:
The ordering of the handlers is mandatory now. It will not run in any other order.